src/main/java/core/basesyntax/dao/writeReportImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Account;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class writeReportImpl implements writeReport {
```

Comment:
The interface name `writeReport` does not follow Java naming conventions. Interface names should be in PascalCase, such as `WriteReport`.